python/lsst/daf/butler/core/datastoreRecordData.py

```diff
@@ -131,12 +182,24 @@ def to_simple(self, minimal: bool = False) -> SerializedDatastoreRecordData:
         simple : `dict`
             Representation of this instance as a simple dictionary.
         """
-        ref_ids = [ref.id for ref in self.refs]
-        record_data = {
-            table_name: [record.to_simple() for record in records]
-            for table_name, records in self.records.items()
-        }
-        return SerializedDatastoreRecordData(ref_ids=ref_ids, records=record_data)
+
+        def _class_name(records: list[StoredDatastoreItemInfo]) -> str:
+            """Get fully qualified class name for the records. Empty string
+            returned if list is empty. Exception is raised if records are of
+            different classes.
+            """
+            if not records:
+                return ""
+            classes = set(record.__class__ for record in records)
```

Comment:
Probably better to use `lsst.utils.introspection.get_full_type_name`, as that tries to determine a "less private" name by dropping leading-underscore modules that we don't want to bake into files.

Reply:
This line is to only make sure that there is a single class for all records, I think it does not matter here what is the exact name and I tried to avoid building a complete class name for each record. `get_full_type_name` is used below to return the name of that single class.